README.md

```diff
@@ -1 +1,22 @@
-# one-click-installers
+# GleSYS one-click installers [![License: CC0-1.0](https://img.shields.io/badge/License-CC0%201.0-lightgrey.svg)](http://creativecommons.org/publicdomain/zero/1.0/)
+> The scripts that powers the one-click installers on GleSYS Cloud (cloud.glesys.com).
+
+## Guidelines
+
+1. The one-click installer should be placed in a folder which is named after a rundown of what it does. Eg. `wordpress-nginx-mariadb`.
+2. The folder should include a `cloudconfig.yaml`, which should include the cloud-init code. And a `README.md`, which should include a summary of what it does.
```

Comment:
In the cloud-init documentation as well as in the examples "cloud config" is always written as `Cloud config` or `cloud-config`. I think it would make sense to name the file `cloud-config.yml` (also matches the `#cloud-config` prefix within the file).